### ISSUE TITLE
unable to convert to json

### ISSUE THREAD

**Issue by hodiapa** [Keyang/node-csvtojson#53]
```javascript

var fs = require("fs");
var Converter = require("csvtojson").Converter;
var csvFileName="./Sacramentorealestatetransactions.csv"
// Source http://samplecsvs.s3.amazonaws.com/Sacramentorealestatetransactions.csv

self = module.exports ={
    getCsvAsJSON: function (callback) {
    //Converter Class
    var fileStream = fs.createReadStream(csvFileName);
    //new converter instance
    var converter = new Converter({ constructResult: true });
    //end_parsed will be emitted once parsing finished
    converter.on("end_parsed", function (jsonObj) {
        console.log(jsonObj); //here is your result json object
        callback(jsonObj);
    });
    //read from file
    fileStream.pipe(converter);
   }

}

self.getCsvAsJSON(function(){})

```
Output I get is always empty
version:  latest

**Comment by Keyang** [comment]
Hi,
the csv data contains Microsoft line breaks "\r"
try initialise csv converter with "eol" parameter like below:
var converter = new Converter({ constructResult: true ,eol:"\r"});

This should fix your issue.

The converter should automatically pick the line breaks. I will fix this
later.

~Keyang

**Comment by Keyang** [comment]
fixed in 4679e7864316f1bb7d31bd58ae611cbf061c1abf
